get_together/templates/get_together/events/show_event.html

```diff
@@ -9,6 +9,14 @@
   <meta property="og:title"         content="{{event.name}}" />
   <meta property="og:description"   content="{{event.summary}}" />
   <meta property="og:image"         content="{{event.team.full_img_url}}" />
+  <!-- AddEvent script -->
+  <script type="text/javascript" async defer>
+  
+    
+    console.log('date', ('{{event.local_start_time}}'));
+    // console.log('formatDate', formatDate('{{event.local_start_time}}'));
+  </script>
+<script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js" async defer></script>
```

Comment:
Non-blocking FYI:

Ultimately, we will want to host all Javascript ourselves. Third-party JS makes our frontend less likely to work with things like uBlock/uMatrix and also opens our users to external tracking. In the long run, we may need to push emergency fixes for these as well (for security, etc).

We need to establish a pattern here and a mechanism for vendoring. I've thrown a story for this on the icebox.